thoth/license_solver/cli.py

```diff
@@ -67,23 +187,49 @@ def cli(_: click.Context, directory: str, file: str, verbose: bool = False) -> N
 
     license_solver = Solver()
 
-    if directory and file:
-        _LOGGER.error("Can't be directory and file parsed at same time.")
-        print("Can't be directory and file parsed at same time. Choose only one", file=sys.stderr)
+    # package argument
+    if package_name is not None:
+        if len(package_name) > 1:
+            if package_version:
+                _LOGGER.warning("Can't insert version to multiple package_name entry")
+                print("Can't insert version to multiple package_name entry", file=sys.stderr)
 
+            for pn in package_name:
+                license_solver.solve_from_pypi(pn, package_version)
+
+        elif len(package_name) == 1:
+            license_solver.solve_from_pypi(package_name[0], package_version)
+        elif package_version:
+            print(ctx.get_help(), "\n\n--package-version is used with --package-name.", file=sys.stderr)
+            exit(1)
+
+    # directory argument
     if directory:
-        if not os.path.isdir(directory):
-            _LOGGER.warning("You need to insert valid directory.")
-            return
+        for d in directory:
+            if not os.path.isdir(d):
+                _LOGGER.warning("Not a valid directory %r", d)
+                print(f'Not a valid directory "{d}" [SKIPPED]')
+                continue
 
-        _LOGGER.debug("Parsing directory: %s", directory)
-        license_solver.solve_from_directory(directory)
-        license_solver.print_output()
+            _LOGGER.debug("Parsing directory: %s", d)
+            license_solver.solve_from_directory(d)
 
+    # file argument
     if file:
-        _LOGGER.debug("Parsing file: %s", file)
-        license_solver.solve_from_file(file)
-        license_solver.print_output()
+        for f in file:
+            if not os.path.isfile(f):
+                _LOGGER.warning("Not a valid file %r", f)
+                print(f'Not a valid file "{f}" [SKIPPED]')
```

Comment:
I'm unsure about these `print` statements. If the logger is setup properly, they should not be needed and users can adjust verbosity based on their requirements. Do you use `init_logging` from thoth-common?